go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/mailgun/holster/v4
 
-go 1.20
+go 1.21
```

Comment:
https://github.com/mailgun/holster/pull/186#discussion_r1502649820

Reply:
Are we ready to drop 1.20 now?  Legacy codebases would have to stay at legacy `holster` releases.  Otherwise, we won't be able to update OTel, which is now set at 1.21.

e.g.:
```sh
$ go get go.opentelemetry.io/otel
go: upgraded go 1.20 => 1.21
go: added toolchain go1.21.5
go: upgraded github.com/go-logr/logr v1.3.0 => v1.4.1
go: upgraded github.com/stretchr/objx v0.5.0 => v0.5.2
go: upgraded github.com/stretchr/testify v1.8.4 => v1.9.0
go: upgraded go.opentelemetry.io/otel v1.21.0 => v1.25.0
go: upgraded go.opentelemetry.io/otel/metric v1.21.0 => v1.25.0
go: upgraded go.opentelemetry.io/otel/trace v1.21.0 => v1.25.0
```

And since 1.20 is now EOL, we should be justified.  https://endoflife.date/go

Reply:
If existing services can update without issue then dropping 1.20 is fine. GH Code Search to find the offenders and notify as necessary 👍